### ISSUE TITLE
Hyper-parameters for reproducing PoseAug on H3.6M

### ISSUE THREAD

**Issue by yangchris11** [jfzhang95/PoseAug#18]
Thank you for your interesting and exciting work!

After going over the code, I wonder if you can also provide the hyper-parameters (total epoch, learning rate ...) to reproduce the result in Table 4, namely the PoseAug using different keypoints source on H3.6M?

Thank you!

**Comment by Garfield-kh** [comment]
Hi, thank you for the interest!

There is some modification in the released code in the BA operation (fixed backbone) and the feedback loss (allowed the hardness within a proper range), which helps achieve better performance for the highest one in 3DHP (vpose-gt: 71) and H36M (stgcn-gt:36).

To reproduce the result in Table 4, you may try with the original Hyper-parameters setting and code [here](https://drive.google.com/file/d/1Y1-u_GhbbmvHapAQPTJUsm4VB7tZTNJU/view?usp=sharing).

The folder structure is the same, just put the `data` folder from the released one to the original one and run the script in `script_final.sh.`

**Comment by yangchris11** [comment]
Thank you so much!
